src/btree/bt_handle.c

```diff
@@ -409,32 +409,18 @@ __btree_conf(WT_SESSION_IMPL *session, WT_CKPT *ckpt)
         btree->checksum = CKSUM_UNCOMPRESSED;
 
     /* Debugging information */
-    WT_RET(__wt_config_gets(session, cfg, "assert.commit_timestamp", &cval));
+    WT_RET(__wt_config_gets(session, cfg, "assert.write_timestamp", &cval));
     btree->assert_flags = 0;
-    if (WT_STRING_MATCH("always", cval.str, cval.len))
-        FLD_SET(btree->assert_flags, WT_ASSERT_COMMIT_TS_ALWAYS);
-    else if (WT_STRING_MATCH("key_consistent", cval.str, cval.len))
-        FLD_SET(btree->assert_flags, WT_ASSERT_COMMIT_TS_KEYS);
-    else if (WT_STRING_MATCH("never", cval.str, cval.len))
-        FLD_SET(btree->assert_flags, WT_ASSERT_COMMIT_TS_NEVER);
-
-    /*
-     * A durable timestamp always implies a commit timestamp. But never having a durable timestamp
-     * does not imply anything about a commit timestamp.
-     */
-    WT_RET(__wt_config_gets(session, cfg, "assert.durable_timestamp", &cval));
-    if (WT_STRING_MATCH("always", cval.str, cval.len))
-        FLD_SET(btree->assert_flags, WT_ASSERT_COMMIT_TS_ALWAYS | WT_ASSERT_DURABLE_TS_ALWAYS);
-    else if (WT_STRING_MATCH("key_consistent", cval.str, cval.len))
-        FLD_SET(btree->assert_flags, WT_ASSERT_DURABLE_TS_KEYS);
-    else if (WT_STRING_MATCH("never", cval.str, cval.len))
-        FLD_SET(btree->assert_flags, WT_ASSERT_DURABLE_TS_NEVER);
+    if (WT_STRING_MATCH("all", cval.str, cval.len))
```

Comment:
The new configuration options are being parsed out in `\_\_conn_dhandle_config_parse`, but I decided to leave these options here for now. I think we should eventually move them to the data handle as well, but not as part of this change.